### PR TITLE
aha/BRacketFix

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/tagliatelle/compiler/Compiler.java [hunk 1]
```diff
@@ -45,6 +45,15 @@
     private static List<ExpressionHandler> expressionHandlers;
 
     /**
+     * The properly handle nested blocks of curly brackets (e.g. JavaScript) interleaved with
```
Comment (Tobsucht) [inline src/main/java/sirius/tagliatelle/compiler/Compiler.java]: The -> To

File: src/main/java/sirius/tagliatelle/compiler/Compiler.java [hunk 2]
```diff
@@ -45,6 +45,15 @@
...
+     * tagliatelle code, we keep track of the total number of open curly brakets.
```
Comment (Tobsucht) [inline src/main/java/sirius/tagliatelle/compiler/Compiler.java]: brackets

File: src/main/java/sirius/tagliatelle/compiler/Compiler.java [hunk 3]
```diff
@@ -45,6 +45,15 @@
...
+     * <p>
+     * Then, when parsing a block of text, we know if we hit a closing bracket if this is the end of the block,
+     * or part of the internal static text.
+     */
+    private int numberOfOpenCurlyBrackets;
```
Comment (Tobsucht) [inline src/main/java/sirius/tagliatelle/compiler/Compiler.java]: = 0 ?

File: src/main/java/sirius/tagliatelle/compiler/Compiler.java [hunk 4]
```diff
@@ -514,26 +527,27 @@ public Expression parseExpression(boolean skipWhitespaces) {
      * Consumes all static template text until either the end of the current block is reached or an expression or tag is
      * started.
      *
+     * @param expectedNumberOfOpenCurlyBrackets the number of open brakets around this block, to known when we hit the
+     *                                          end of the block when lloking for a }
```
Comment (fkrone) [inline src/main/java/sirius/tagliatelle/compiler/Compiler.java]: looking - wrong spelling

**Pull request merged**